dp/generate_climos.py

```diff
@@ -398,15 +419,20 @@ def main(args):
 if __name__ == '__main__':
     parser = ArgumentParser(description='Create climatologies from CMIP5 data')
     parser.add_argument('filepaths', nargs='*', help='Files to process')
-#    parser.add_argument('-c', '--climo', nargs= '+',  help='Climatological periods to generate. IN PROGRESS. Defaults to all available in the input file. Ex: -c 6190 7100 8100 2020 2050 2080')
+    # parser.add_argument('-c', '--climo', nargs= '+',  help='Climatological periods to generate.
+    # IN PROGRESS. Defaults to all available in the input file. Ex: -c 6190 7100 8100 2020 2050 2080')
     log_level_choices = 'NOTSET DEBUG INFO WARNING ERROR CRITICAL'.split()
     parser.add_argument('-l', '--loglevel', help='Logging level',
                         choices=log_level_choices, default='INFO')
     parser.add_argument('-n', '--dry-run', dest='dry_run', action='store_true')
-    parser.add_argument('-g', '--convert-longitudes', dest='convert_longitudes', action='store_true')
-    parser.add_argument('-s', '--split-vars', dest='split_vars', action='store_true')
+    parser.add_argument('-g', '--convert-longitudes', dest='convert_longitudes', action='store_true',
```

Comment:
One thing that I notice about all of these arguments (convert-longitudes, split-vars and split-intervals), is that they are False by default. However each of them are essentially represent "do something that is the best practice. So by default we're doing the *wrong* thing, and making the user explicitly ask the script to do the *right* thing.

Changing the behaviour  would probably mean changing the default to ``True``, removing the ``action`` parameter, changing the type to bool and making the user say something like ``--split-vars=False``.